script/timelocked/UpgradeFeedManagerAndAdaptersTimelocked.s.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.25;
+
+import { Upgrades } from "openzeppelin-foundry-upgrades/Upgrades.sol";
+import { ITransparentUpgradeableProxy } from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import { ProxyAdmin } from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import { UpgradeableBeacon } from "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
+import { EOFeedFactoryBeacon } from "../../src/adapters/factories/EOFeedFactoryBeacon.sol";
+import { Script } from "forge-std/Script.sol";
+import { stdJson } from "forge-std/Script.sol";
+import { EOJsonUtils } from "../utils/EOJsonUtils.sol";
+import { TimelockController } from "@openzeppelin/contracts/governance/TimelockController.sol";
+import { TimelockBase } from "./TimelockBase.sol";
+import { EOFeedManager } from "../../src/EOFeedManager.sol";
+import { EOFeedAdapterV2 } from "../../src/adapters/EOFeedAdapterV2.sol";
```

Comment:
in internal audit PR - we agreed to change name of the EOFeedAdapterV2 to more evident, pointing the purpose to support compatibility of all old adapters . 
it is changed to EOFeedAdapterOldCompatible
https://github.com/Eoracle/target-contracts/pull/197/files